react/DisplayButtonTypes.js

```diff
@@ -0,0 +1,26 @@
+import { values, map } from 'ramda'
+
+const displayButtonTypes = {
+  DISPLAY_ALWAYS: {
+    name: 'editor.productSummary.displayBuyButton.option.always',
+    value: 'displayButtonAlways',
+  },
+  DISPLAY_ON_HOVER: {
+    name: 'editor.productSummary.displayBuyButton.option.hover',
+    value: 'displayButtonHover',
+  },
+  DISPLAY_NONE: {
+    name: 'editor.productSummary.displayBuyButton.option.none',
+    value: 'displayButtonNone',
+  },
+}
+
+export function getDisplayButtonNames() {
+  return map(opt => opt.name ,values(displayButtonTypes))
```

Comment:
The comma must be right after a word and not right before the next one according to eslint restrictions. :pick: